Assets/Scripts/WorkPuzzle/WorkInput.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WorkInput : MonoBehaviour
+{
+    public WorkManager workManager;
+
+    /*
+     * Tracks the characters being clicked on the keyboard using an input string; 
+     * characters are to be passed on to the TypeKey method of WorkManager
+     */
+    void Update()

```

Comment:
This could be a good place for this but maybe use a coroutine so we can start and stop this easily?